src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 24.9.1. [TC-LVL-9.1] Dependency with the On/Off cluster Verification (DUT as Server)
```

Comment:
I don't see a TC-LVL-9.1 in https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/levelcontrol.adoc

If this is not an actual cert test, it should be in just `src/app/tests/suites` and have a name that does not look like a certification test.

This test name also needs to be added to `src/app/tests/suites/tests.js` and then you want to run:
```
./scripts/tools/zap/generate.py src/controller/data_model/controller-clusters.zap -t examples/darwin-framework-tool/templates/tests/templates.json -o zzz_generated/darwin-framework-tool/zap-generated
```
and
```
./scripts/tools/zap/generate.py -t examples/chip-tool/templates/tests/templates.json -o zzz_generated/chip-tool/zap-generated src/controller/data_model/controller-clusters.zap
```
to regenerate the generated tests.

Reply:
I've moved the test in `src/app/tests/suites/tests.js`, under the name `TestLevelControlWithOnOffDependency`, and regenerated the tests.